lists/fi/fi-pro.json

```diff
@@ -38,7 +38,7 @@
+        "en": "All business operating in Finland must obtain a Business ID (\"Y-tunnus\" in Finnish) from the Finnish Trade Register. The Finnish Trade Register is maintained by the Finnish Patent and Registration Office (PRH), whom are also responsible for maintaining the Business Information System, which can be used to search for all companies in Finland, and the API which allows users to download company information in bulk. \n\n\"The Finnish Trade Register (Finnish: Kaupparekisteri, Swedish: Handelsregistret) is a company register in Finland. It provides official information on businesses in the whole country, including data from current and old register entries, articles of association, partnership agreements or rules.\" [1]\n\n\"The Finnish Trade Register is a public register that contains information about businesses and companies. As a rule, all businesses have to be registered at the Trade Register. Businesses also have to notify the register of any changes in their registered details. Most businesses must also submit their financial statements (annual accounts) to the register. \" [2] \n\n\"The Business Information System BIS (\"YTJ\" in Finnish) is a service jointly maintained by the Finnish Patent and Registration Office and the Finnish Tax Administration, enabling you to file information to both authorities using one single notification. \n\nYou can use the BIS to:\n\nstart a business or an organization\nreport changes \nclose down a business or an organization\nsearch for basic details of companies and organizations using the company search\" [3]\n\n[1] https://en.wikipedia.org/wiki/Finnish_Trade_Register\n[2] https://www.prh.fi/en/kaupparekisteri/rekisterointipalvelut.html\n[3] https://www.ytj.fi/en/index/whatisbis.html"
```

Comment:
"...(PRH), whom are also responsible..."

prob better as 

"...(PRH), which is also responsible..."

Reply:
Done in https://github.com/org-id/register/pull/459/commits/1616fc1bc102223c717a301ca888fb3270797c05